layouts/_default/single.html

```diff
@@ -30,6 +30,7 @@ <h1 class="post-title">
   <div class="post-content">
     {{ .Content }}
   </div>
+  {{ if not .Site.Params.disableReadOthers }}
```

Comment:
If this won't work after changes above, I'd change it to:

```suggestion
  {{ if ne .Site.Params.disableReadOtherPosts true }}
```